concert/ext/ufo.py

```diff
@@ -523,9 +576,14 @@ class GeneralBackprojectManager(Parameterizable):
 
     state = State(default='standby')
 
-    async def __ainit__(self, args, average_normalization=True, regions=None, copy_inputs=False):
+    _readiness: asyncio.Event
+
+    async def __ainit__(self, args: GeneralBackprojectArgs, readiness: asyncio.Event,
```

Comment:
1. readiness is a weird name; 2. it must be a keyword parameter because for offline reconstruction it is useless.